### ISSUE TITLE
Inherited fields cannot be used for as paginationKeys 

### ISSUE THREAD

**Issue by haydnnewport** [benjamin658/typeorm-cursor-pagination#10]
When passing a query to be paginated, if `paginationKeys` contains an inherited field, it causes an error at https://github.com/benjamin658/typeorm-cursor-pagination/blob/155811a4b487f9b37dca617873417d9039adebbb/src/Paginator.ts#L199
with the signature `TypeError: Cannot read property 'name' of undefined`.

This seems to be because this line is using `getOwnMetadata` as opposed to `getMetadata`. Is there a particular reason for this to use that function?

Demonstration code:
```typescript
@ArgsType()
@ObjectType()
export abstract class EntityBase {
  @PrimaryGeneratedColumn('uuid')
  @Field((type) => ID, { nullable: true }) // nullable for mutations creating one
  readonly id!: string;

  @CreateDateColumn({ type: 'timestamp' })
  createdAt!: Date;

  @UpdateDateColumn({ type: 'timestamp' })
  updatedAt!: Date;
}

export class Service extends EntityBase {
  @Field()
  @Column({ length: FieldSpecs.SINGLE_LINE })
  @IsString()
  name!: string;
}
```

```typescript
const paginator = buildPaginator({
  entity: Service,
  ["createdAt"],
  query: {limit: 10, order: "ASC"},
});
const paginated = await paginator.paginate(query);
```

**Comment by benjamin658** [comment]
Hi @haydnnewport , thanks for reporting the issue.

> Is there a particular reason for this to use that function?

Actually no, it just because I forgot the entity inheritance use case when developing this library, and it should work properly with `getMetadata` too. 

It will be fixed in the next release.

**Comment by benjamin658** [comment]
Hi @haydnnewport , 

The issue has been fixed, please upgrade to v0.4.1.
